src/sentry/tasks/post_process.py

```diff
@@ -468,10 +481,38 @@ def post_process_group(
                 return
             # Issue platform events don't use `event_processing_store`. Fetch from eventstore
             # instead.
-            event = eventstore.get_event_by_id(
-                project_id, occurrence.event_id, group_id=group_id, skip_transaction_groupevent=True
+
+            def get_event_with_retry(
+                getter_func: Callable[[], Optional[Event]],
+                retry_exceptions: Sequence[Type[Exception]] = (),
+                attempts=3,
+            ) -> Optional[Event]:
+                attempt = 1
+                delay_exponent = 0
+                event_or_none = None
+                while event_or_none is None or attempt <= attempts:
```

Comment:
We have `TimedRetryPolicy` that is similar here: https://github.com/getsentry/sentry/blob/c4dbed6ff83745c2e15351569b7632ec75265830/src/sentry/utils/retries.py#L117. It doesn't use exponential backoff, but you could pass a function to `delay` to do that instead

Reply:
Ohh, I was trying to find a utility function for retries. Thanks